build.gradle

```diff
@@ -49,6 +48,6 @@ allprojects {
 }
 
 dependencies {
-    api 'com.apptentive:apptentive-kit-android:6.0.3'
+    implementation 'com.apptentive:apptentive-kit-android:6.1.0'
```

Comment:
Must remain at api instead of implementation

Reply:
Changed back to api